### ISSUE TITLE
Usefulness of the average brain

### ISSUE THREAD

**Issue by mrjiaruiwang** [openconnectome/Statistical-Connectomics-Class#147]
What are the good and bad of using an "average brain" for analysis? Given the massive genetic and phenotypical deviation between individuals, it would seem almost foolish to compare our models to some average. I can get the ball rolling:

Pros:
- Something is better than nothing
- Gives experiments a point of reference to compare to

Cons:
- Different from pretty much every brain out there
- Maps are inaccurate because people's brains change over time

**Comment by whock** [comment]
Yeah, I agree that the high inter-subject variability is something to take into account. To answer how useful "averages" can be when there is such massive, and highly variable, data, it might help to look to similar examples in biology. One would be genetics. The human genome is enormous as well as possessing significant variation from person to person. But we can still make inferences based on the "average" genome, in spite or, or even sometimes because of, this variability. For example there are SNPs (a single base pair that commonly varies within the population). We can sometimes make useful diagnostic predictions about what disease/abnormality someone has by comparing their SNP profile to that of the average. So even though the "average" connectome doesn't even exist for most of us (i.e. who actually *has* the connectome specified by the average? Maybe nobody.) we can still analyze how individuals vary from the average and make diagnostic inferences based on this.

**Comment by akim1** [comment]
In my view, the "average" connectome is more about identifying the similarities among human brains than about their differences. In the same way that a person lacking a kidney or a lung would be a cause of concern, the "average" connectome would give us a tool to identify people missing essential connections found in the rest of the population. The only problem is that missing organs are easily identifiable just by simple observation, but missing neural connections requires a state of knowledge and technology that we don't quite have yet.

**Comment by dlee138** [comment]
Although "average" can be very useful, as other people have mentioned, but we still have to watch out for obvious differences that may arise between different populations/groups.  The same way genomes may differ between different groups (i.e. XX vs XY for females and males), there may be subtle, but important differences between the average brains of two different groups.  So while average brains are important the average brains for different populations is just as important as well.

**Comment by DSP137** [comment]
We take samples and use the sample average as an estimate for the true average, but in stats (or at least baby-stats), instead of simply saying that the true average is approximately the sample average, we associate some confidence, saying that we are 90% or 95% confident that the true average is within some range of values based on the sample mean and standard error. I know we do this for normal distributions. Is there a way to apply this kind of idea to say that 'normal' is more accurately described as a range of values?

**Comment by mrjiaruiwang** [comment]
So one thing I have been thinking about, going with what whock posted about, is non-parametric testing as applied to connectomics. In genetics, we could compare SNPs to some "average" SNP, but indeed we can perform many tests on the SNPs themselves, such as mutual information, rank sum tests, and permutation tests. None of these methods require some "average" expression. In the case of rank sum, you do not even have to assume the probability distribution. While I do support the mean connectome, I also support using other tests like permutation testing with a test statistic like the number of clusters, with a Bonferroni correction to maybe find associations between connectomes and phenotypes such as disease prognosis, treatment effectiveness, or symptoms. I just think that while the construction of an average connectome is an important exercise in our understanding of the brain, it is not necessary for us to begin to think about ways of delivering clinical impact from this framework. We could use the connectome features associated with phenotypes for diagnosis and prediction.
